ICSharpCode.CodeConverter/CSharp/NodesVisitor.cs

```diff
@@ -1855,18 +1855,31 @@ public override CSharpSyntaxNode VisitIdentifierName(VBSyntax.IdentifierNameSynt
                                                     || node.Parent is VBSyntax.MemberAccessExpressionSyntax maes && maes.Expression == node
                                                     || node.Parent is VBSyntax.QualifiedNameSyntax qns && qns.Left == node
                     ? QualifyNode(node, identifier) : identifier;
-                return AddEmptyArgumentListIfImplicit(node, qualifiedIdentifier);
+
+                var withArgList = AddEmptyArgumentListIfImplicit(node, qualifiedIdentifier, out bool addedArgumentList);
+                if (!addedArgumentList && !node.Parent.IsKind(VBasic.SyntaxKind.InvocationExpression)) {
+                    var vbMethodBlock = node.Ancestors().OfType<VBSyntax.MethodBlockSyntax>().FirstOrDefault();
```

Comment:
I think this needs to be `MethodBlockBaseSyntax` to account for this case:
https://dotnetfiddle.net/FpGqfE

Reply:
The symbol thing looks much better, I think you still need to switch to `OfType<MethodBlockBaseSyntax>`  here though.

Reply:
Ah I see, this is a separate issue and we can address it separately (https://github.com/icsharpcode/CodeConverter/issues/295)